src/Storefront/Resources/views/storefront/layout/navigation/active-styling.html.twig

```diff
@@ -0,0 +1,36 @@
+{% block layout_navigation_active_styling %}
```

Comment:
What is this used for? Some inline comments would be nice, too. 😁

Reply:
added some comments, please have a look, if it is clearer now